### DISCUSSION TITLE
Remove "Learn about something" from Control Panel

### DISCUSSION THREAD

**Discussion by WetDry** [statamic/cms#5614] [Q&A]
Hello guys!
I've a very simple question. There is a way to remove the label "Learn about something" (where something is the name of current section of control panel) ? I'm asking because i've a client that want a very very clean Control Panel without any distraction.

Thank you all for information!

**Comment by jackmcdade** [comment]
There sure is! 

https://statamic.dev/white-labeling#documentation

**Reply by WetDry** [reply]
Thank you so much Jack!
